.papr.yml

```diff
@@ -91,6 +91,21 @@ tests:
 
 ---
 
+inherit: true
+required: true
+
+context: f26-introspection-tests
+
+env:
+    CFLAGS: ''  # Reset ASAN
```

Comment:
By respecifying `env` it effectively replaces the previous dict, so you shouldn't need that. Might want to leave it to be more explicit though.